src/domains/pay-wallet-app/05_aks_middleware_tools.tf

```diff
@@ -0,0 +1,53 @@
+module "tls_checker" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//tls_checker?ref=v6.3.0"
+
+  https_endpoint                         = local.payment_wallet_hostname
+  alert_name                             = local.wallet_hostname
```

Comment:
sorry, my bad, in previous suggestion I linked only one variable, `wallet_hostname` to `payment_wallet_hostname` should be applied also here and in locals file, right?

Reply:
fixed